atecc608a_se.c

```diff
@@ -15,6 +15,37 @@
 #endif
 #include <stdbool.h>
 
+#define ATCAB_INIT()                                        \
+    do                                                      \
+    {                                                       \
+        if (atcab_init(&atca_iface_config) != ATCA_SUCCESS) \
+        {                                                   \
+            status = PSA_ERROR_HARDWARE_FAILURE;            \
+            goto exit;                                      \
+        }                                                   \
+    } while(0)
+
+#define ATCAB_DEINIT()    \
+    do                    \
+    {                     \
+        atcab_release();  \
+    } while(0)
+
+#define ASSERT_STATUS(expression, expected, result)             \
+    do                                                          \
+    {                                                           \
+        status = (expression);                                  \
+        int ASSERT_STATUS_expected = (expected);                \
```

Comment:
`status` is an `int32_t` storing PSA errors. `expected` is an `int` storing ATCA errors. You can't meaningfully compare the two, so `status = (expression)` doesn't make sense.

Reply:
I've made a mistake while writing this - please see the new version. Thanks!